tests/sesolve/test_batching.py

```diff
@@ -73,18 +58,16 @@ def test_timearray_batching():
 
     # == modulated time array
     deltas = jnp.linspace(0.0, 1.0, 4)
-    H_mod = dq.modulated(lambda t, delta: jnp.cos(t * delta), H0, args=(deltas,))
+    H_mod = dq.modulated(lambda t: jnp.cos(t * deltas), H0)
 
     result = dq.sesolve(H_mod, psi0, times)
     assert result.states.shape == (4, 11, 4, 1)
-    result = dq.sesolve(H0 + H_mod, psi0, times)
-    assert result.states.shape == (4, 11, 4, 1)
+    # result = dq.sesolve(H0 + H_mod, psi0, times)
+    # assert result.states.shape == (4, 11, 4, 1)
```

Comment:
Is time-array addition broken?

Reply:
No, it's a reminiscence from darker times